google/pubsub/v1/pubsub_gapic.yaml

```diff
@@ -168,17 +176,29 @@ interfaces:
     retry_codes:
     - UNAVAILABLE
     - DEADLINE_EXCEEDED
+  - name: one_plus_delivery
```

Comment:
It happens to be the same now, but I'm thinking that these two cases might differ in the future.